code/src/java/pcgen/core/SettingsHandler.java

```diff
@@ -806,7 +796,6 @@ public static void getOptionsFromProperties(final PlayerCharacter aPC)
 				.parseInt(dw.toString().substring(0, Math.min(dw.toString().length(), dw.toString().lastIndexOf('.'))));
 			final int height = Integer
 				.parseInt(dh.toString().substring(0, Math.min(dh.toString().length(), dh.toString().lastIndexOf('.'))));
-			d = new Dimension(width, height);
```

Comment:
Wait, you remove Dimension, but not what leads to it?

Reply:
good call.